### DISCUSSION TITLE
Only health check have tracking data, but other endpoints do not. What is the possible reason?

### DISCUSSION THREAD

**Discussion by xiangzirui** [apache/skywalking#12143] [Q&A] [answered]
Service Image Version:
elasticsearch:8.4.0
apache/skywalking-ui:9.7.0
apache/skywalking-oap-server:9.7.0
Java agent:
version: 9.1.0
VM Options: -javaagent:skywalking-agent.jar=agent.service_name=group::service-name,collector.backend_service=skywalking-oap.skywalking.svc:11800)
Status quo:
![image](https://github.com/apache/skywalking/assets/49627024/ee956e13-e190-4b60-b62b-328984491545)

**Comment by wu-sheng** [comment, marked as answer]
They may have one or two access hours ago, but not your selected duration.

Can't tell the exact reason.
